crates/apps/Cargo.toml

```diff
@@ -37,12 +37,6 @@ doc = false
 name = "namadaw"
 path = "src/bin/namada-wallet/main.rs"
 
-# Namada relayer
-[[bin]]
-doc = false
-name = "namadar"
-path = "src/bin/namada-relayer/main.rs"
-
```

Comment:
isn't this change alone sufficient to disable the build? without any other changes to rust source files

Reply:
I tried that at first but then namada still built it, just with the name `namada-relayer` instead of `namadar`

Reply:
[0001-Disable-the-Namada-relayer.patch](https://github.com/user-attachments/files/16886236/0001-Disable-the-Namada-relayer.patch)

Try applying these changes over a clean `main`

Reply:
thx this works. Changed